packages/api/src/domain/medical/document-bulk-download.ts

```diff
@@ -0,0 +1,24 @@
+export const DocumentDownloadStatus = ["processing", "completed", "failed"] as const;
+export type DocumentDownloadStatus = (typeof DocumentDownloadStatus)[number];
+
+export type Progress = {
+  status: DocumentDownloadStatus;
+  total?: number;
+  successful?: number;
+  errors?: number;
+};
+
+export type DocumentBulkDownloadProgress = {
```

Comment:
modelled after `DocumentQueryProgress`

Reply:
To Dima's point (on the docs, it seems like we can simplify this and have only the req ID and the status?

Diff from DQ/DR, this process is unlikely to error since its just generating signed URLs for existing docs.